src/utility/PricingStrategy.js

```diff
@@ -3,6 +3,7 @@ const priceData = {
   fixed_price_per_method: "fixed_price_per_method",
   agi_precision: 100000000,
   agi_divisibility: 8,
+  usd_conv_rate: 41.925594,
```

Comment:
Should we be pulling this information from some service?

Reply:
This is a dummy rate right now.@raam said he'll confirm on this one.

Reply:
👍